app/components/peopletable/peopletable.js

```diff
@@ -145,13 +145,13 @@ const PeopleTable = translate()(class PeopleTable extends React.Component {
     WindowSizeListener.DEBOUNCE_TIME = 50;
   }
 
-  handleEmptySearch(e){
+  handleEmptySearch(){
     this.handleFilterChange(null);
   }
 
   componentDidMount() {
     //setup default sorting but don't track via metrics
-    this.handleSortChange('fullNameOrderable', SortTypes.ASC, false);
+    this.handleSortChange('lastNameOrderable', SortTypes.ASC, false);
```

Comment:
Looks strange to me to have a default two columns sort that you can't reproduce through the UI.

Reply:
Yes and no.
I think a sort by name, should sort the other part (first / last) too.
If there is 2 person with the same lastname for example, having them sort by firstname seems OK to me.

Also we have actually a problem in that list, the order is not always the same, when display it (without selecting a sort) this will fix that probably.

Reply:
Disagreed. It requests an additional cognitive task from the user (pick the field where they should type in). On any website you only have one search box.